### PR TITLE
SPL-464 - Update cookie banner

### PR DESCRIPTION
**Pull request by devind-ra**

### CODE REVIEW

File: app/views/layout.njk [hunk 1]
```diff
@@ -9,22 +10,47 @@
...
-      <button value="accept" type="button" name="cookies" class="govuk-button">Accept all cookies</button>
-      <a class="govuk-button" href="/cookies">Set cookie preferences</a>
+  <header id="cookie_banner" class="cookie-banner">
+      {% set html %}
+        <h1 class="govuk-heading-m">Cookies on {{ service_name }}</h1>
+        <p>We use some essential cookies to make this service work.</p>
+        <p>We’d like to set additional cookies so we can remember your settings, understand how people use the service and make improvements.</p>
+      {% endset %}
+      {% set acceptHtml %}
+        <p>You’ve accepted additional cookies. You can <a class="govuk-link" href="/cookies">change your cookie settings</a> at any time.</p>
```
Comment (Louis-Solirius) [inline app/views/layout.njk:20]: I'm not sure this is working, when I click accept or reject nothing comes up, when this looks like it should make something come up

File: app/views/layout.njk [hunk 2]
```diff
@@ -9,22 +10,47 @@
...
-    <header id="cookie_banner" class="cookie-banner">
-      <h1 class="govuk-heading-l">Tell us whether you accept cookies</h1>
-      <p>We use <a href="/cookies">cookies to collect information</a> about how you use the {{
-        service_name }} service. We use this information to make the website work as well as
-        possible and improve government services.</p>
-      <button value="accept" type="button" name="cookies" class="govuk-button">Accept all cookies</button>
-      <a class="govuk-button" href="/cookies">Set cookie preferences</a>
+  <header id="cookie_banner" class="cookie-banner">
+      {% set html %}
+        <h1 class="govuk-heading-m">Cookies on {{ service_name }}</h1>
```
Comment (Louis-Solirius) [inline app/views/layout.njk:15]: Where is `service_name` defined? (Insanely nit-picky) but can we change the text to be upper case?

File: app/views/layout.njk (hunk 1 above)
Comment (devind-ra) [reply app/views/layout.njk:20]: ah yes with that I was trying to implement a confirmation message after you reload the website but I could't implement it. Will remove it for now

File: app/views/layout.njk (hunk 2 above)
Comment (devind-ra) [reply app/views/layout.njk:15]: I think it was from the file but I'm not too sure - https://github.com/UKGovernmentBEIS/beis-spl-planner/blob/master/server.js

Comment (Louis-Solirius) [reply app/views/layout.njk:15]: it's in app/constants.js. @lambley can we change `SERVICE_NAME: 'Plan shared parental leave and pay'` to `SERVICE_NAME: 'Plan Shared Parental Leave and Pay'`?

Comment (lambley) [reply app/views/layout.njk:15]: I've changed this SERVICE_NAME to title case

**Review by lambley — CHANGES_REQUESTED** [review]
Padding issue on the cookie banner needs to be fixed

**Review by lambley — APPROVED** [review]

**Pull request merged**